src/milkdrop.js

```diff
@@ -23,4 +23,102 @@ export const milkdropOptions = {
 	},
 };
 
+const block = document.querySelector( '.wp-block-tenup-winamp-block' );
+
+var posEqu = block?.dataset.posequ || "1";
+var posList = block?.dataset.poslist || "2";
+var posMilkdrop = block?.dataset.posmilkdrop || "3";
+
+//console.log("posEqu: "+posEqu);
+//console.log("posList: "+posList);
+//console.log("posMilkdrop: "+posMilkdrop);
```

Comment:
Seems like these were added for testing purposes and don't think its necessary for production.